hbase-server/src/main/java/org/apache/hadoop/hbase/io/util/MemorySizeUtil.java

```diff
@@ -234,7 +234,7 @@ public static long getOnHeapCacheSize(final Configuration conf) {
   }
 
   /**
-   * @param conf used to read config for bucket cache size. (< 1 is treated as % and > is treated as MiB)
+   * @param conf used to read config for bucket cache size. 
```

Comment:
Did we remove the percentage facility? i.e. if < 1, the bucket cache is a percentage of heap? We only do hard size in MBs now?

Reply:
From the code and the test results, yes.

Reply:
![](https://issues.apache.org/jira/secure/attachment/13017745/13017745_image-2020-12-28-17-53-32-575.png)

Reply:
Yes Stack.  The bucket cache size can only be specified as absolute MBs value.  we no longer support on heap Bucket cache and so size as % of heap.  And min value for this is 1. (1 MB)